### ISSUE TITLE
The orb shows when display changing resolution in the bootup sequence

### ISSUE THREAD

**Issue by 111ebobo** [xxxman360/LogonVistaOrb#3]
Add a possibility to add the delay

**Comment by xxxman360** [comment]
Could you perhaps describe this a bit better? LogonVistaOrb starts at the same time LogonUI.exe normally would. Do you have a UEFI system by any chance?

**Comment by 111ebobo** [comment]
Yes, i have UEFI system, and it start's before LogonUI.exe. And for the context in my case, when i start my pc it firstly goes in low resolution, and when windows goes to the LogonUI.exe it changes the resolution to whatever i chose

**Comment by xxxman360** [comment]
I wonder if you're that guy from the WinClassic fourms who said "it didn't work on my main".
Alright, I can easily implement a delay config into LogonVistaOrb for version v1.1. It will be disabled by default, as this is a niche case with you, but who knows, it could help someone else out in the future.

**Comment by 111ebobo** [comment]
> I wonder if you're that guy from the WinClassic fourms who said "it didn't work on my main". Alright, I can easily implement a delay config into LogonVistaOrb for version v1.1. It will be disabled by default, as this is a niche case with you, but who knows, it could help someone else out in the future.

no, i'm not that guy, and yesterday i saw the orb when i was shutting down my cp

**Comment by xxxman360** [comment]
Interesting...

**Comment by 111ebobo** [comment]
> Interesting...

Well, it happened only once, also i have disabled fast startup and it sometimes does show up (the last 2 seconds)
